src/command/restore/restore.c

```diff
@@ -1248,9 +1296,13 @@ restoreSelectiveExpression(Manifest *manifest)
             {
                 const String *db = strLstGet(dbList, dbIdx);
 
-                // Only user created databases can be zeroed, never system databases
-                if (cvtZToUInt64(strZ(db)) >= PG_USER_OBJECT_MIN_ID)
+                // Include system databases by default
+                if (strLstExists(excludeDbIdList, db) ||
+                	(strLstSize(includeDbIdList) > 0 && cvtZToUInt64(strZ(db)) >= PG_USER_OBJECT_MIN_ID && 
+                	!strLstExists(includeDbIdList, db) && !strLstExists(systemDbIdList, db)))
                 {
+                    LOG_DETAIL_FMT("database with id %s will be zeroed", strZ(db));
```

Comment:
Is it possible to print database name in addition to database id? Since this is done for system databases: [L1230](https://github.com/pgbackrest/pgbackrest/pull/1307/files#diff-9cfdda13c7fe664728b8fb1c44176c0408e96ac403d1e15e2f96265e5ddfa35fR1230)
The output will become more useful.

Reply:
that's possible yes, it requires to loop through the manifest-db-list. See commit 3e12e9c that adds this info.